airflow/utils/process_utils.py

```diff
@@ -20,14 +20,20 @@
 import errno
 import logging
 import os
-import pty
 import select
 import shlex
 import signal
 import subprocess
 import sys
-import termios
-import tty
+from airflow.utils.platform_utils import is_windows
+
+if is_windows():
+    from airflow.windows_extensions import termios, tty, pty
+else:
+    import pty
+    import termios
+    import tty
```

Comment:
I feel we should have a global shim (maybe a new `airflow.platform` that works similarly to `airflow.compat`) and use it in all places instead of doing `if is_windows()` everywhere.

Reply:
I will do that. If I understand you correctly you basically do import distinction in one file and than just import from there.